### ISSUE TITLE
`@POMDP_require` deprecation in POMDPTools

### ISSUE THREAD

**Issue by WhiffleFish** [JuliaPOMDP/POMDPs.jl#463]
```julia
julia> using POMDPTools
[ Info: Precompiling POMDPTools [7588e00f-9cae-40de-98dc-e0c70c48cdd7]
┌ Warning: POMDPs.@POMDP_require is deprecated, use POMDPLinter.@POMDP_require instead.
└ @ POMDPs ~/code/random/POMDPs.jl/src/deprecated.jl:18
```

**Comment by zsunberg** [comment]
Yeah, this is super annoying. I do not know how to track it down though, ha. I guess we need to register POMDPs.jl 1.0 and remove POMDPs.@POMDP_require fully.

**Comment by WhiffleFish** [comment]
Is there anything stopping us from releasing a POMDPs.jl v0.10?

Anyways, it appears that changing `@POMDP_require` calls to explicit `POMDPLinter.@POMDP_require` calls fixes the issue.

If we're willing to admit this temporary workaround I can open a PR. If not, semver would suggest that moving from 0.9 to 0.10 would admit breaking changes. So technically it would be fine to remove `POMDPs.@POMDP_require` along with `POMDPs.@req`, `POMDPs.@subreq`, etc. in that version bump, no?

**Comment by WhiffleFish** [comment]
fixed by #466
